code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -1035,3 +1035,42 @@
 	color = "#F5F5F5"
 
 // This reagent's effects are handled in heart attack handling code
+
+/datum/reagent/medicine/nanocalcium
+	name = "Nano-Calcium"
+	id = "nanocalcium"
+	description = "Highly advanced nanites equipped with calcium payloads designed to repair bones without surgery suspended in synthetic blood plasma."
+	color = "#9b3401"
+	metabolization_rate = 0.5
+
+/datum/reagent/medicine/nanocalcium/on_mob_life(mob/living/carbon/human/M)
+	switch(current_cycle)
+		if(1 to 19)
+			M.AdjustJitter(4)
+			if(prob(10))
+				to_chat(M, "<span class='notice'>Your skin feels hot and your veins are on fire!</span>")
```

Comment:
Should probably be `span class=danger`, not `span class=notice`.

Reply:
I would actually go with `<span class='warning'>`

Danger is usually used with things that do damage or some sort of tangible negative effect.

Reply:
Edited my comment - I meant that :P